website/content/installation/_index.md

```diff
@@ -110,5 +112,19 @@ secretGenerator:
     - configs/secretkey
 
 generatorOptions:
- disableNameSuffixHash: true
+  disableNameSuffixHash: true
+```
+
+```yaml
+# configs/config
+address-pools:
+  - name: default
+    protocol: layer2
+    addresses:
+      - 192.168.0.128/25
+```
+
+```
+# configs/secretkey
+secretkey=SZlmcghfKLpoj+rnGKcxAeKQY+H/lwo0t10GJ8M3dI8=
```

Comment:
I don't think we should include a sample secret here because inevitably someone will just copy/paste this as-is and have an insecure cluster as a result.  I think just providing instructions on how to generate the secret is better.